PCbuild/pyproject.props

```
@@ -42,6 +42,8 @@
   <PropertyGroup>
     <_DebugPreprocessorDefinition>NDEBUG;</_DebugPreprocessorDefinition>
     <_DebugPreprocessorDefinition Condition="$(Configuration) == 'Debug'">_DEBUG;</_DebugPreprocessorDefinition>
+    <_PyStatsPreprocessorDefinition>PyStats;</_PyStatsPreprocessorDefinition>
+    <_PyStatsPreprocessorDefinition Condition="$(Py_STATS) == ''">Py_STATS;</_PyStatsPreprocessorDefinition>
```

```suggestion
    <_PyStatsPreprocessorDefinition Condition="$(Py_STATS) != ''">Py_STATS;</_PyStatsPreprocessorDefinition>
```

Should this be checking for a non-empty value?